installers/msi-language/CustomFatalError.wxs

```diff
@@ -18,13 +18,13 @@
 				<Control Id="CustomTextFatalError" Type="Hyperlink" X="135" Y="110" Width="220" Height="20" Transparent="yes" Hidden="yes" Text="!(loc.FatalErrorPrimary)">
 					<Condition Action="show">NETWORK_ERROR = "false"</Condition>
 				</Control>
-				<Control Id="CustomTextNetworkError" Type="Text" X="135" Y="70" Width="220" Height="60" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorPrimary)">
+				<Control Id="CustomTextNetworkError" Type="Text" X="135" Y="60" Width="220" Height="60" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorPrimary)">
 					<Condition Action="show">NETWORK_ERROR = "true"</Condition>
 				</Control>
-				<Control Id="CustomTextNetworkErrorDetails" Type="Text" X="135" Y="130" Width="220" Height="40" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorDetails)">
+				<Control Id="CustomTextNetworkErrorDetails" Type="Text" X="135" Y="120" Width="220" Height="60" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorDetails)">
```

Comment:
I wonder if we can actually use a [ScrollableText control](https://docs.microsoft.com/en-us/windows/win32/msi/scrollabletext-control) here, just like in the EULA dialog.  That way we would ensure that we always have enough space for the error text.

Reply:
I spent some time trying different alternatives. The issue with a `ScrollableText` element is that it requires an `.rft` file at build time to populate the text box. Apparently it is possible to open the MSI database at runtime and replace that file, however after some investigation I did not go down that route. Come to think of it I don't think a deferred custom action can access the MSI database as it has to get access via the `session` object.

Reply:
I actually also remember looking into that for some other dialog.  Well, another of those strange limitations.  Let's leave it like this.  There is quite a bit of space after all...